e2e_tests/src/e2e_conduit/scenarios/follower_indexer_scenario.py

```diff
@@ -1,14 +1,108 @@
-import e2e_conduit.fixtures.importers as importers
-import e2e_conduit.fixtures.processors as processors
-import e2e_conduit.fixtures.exporters as exporters
+import time
+
+from e2e_common.indexer_db import IndexerDB
+from e2e_conduit.fixtures import importers, exporters
 from e2e_conduit.scenarios import Scenario
 
 
-def follower_indexer_scenario(sourcenet):
-    return Scenario(
-        "follower_indexer_scenario",
-        importer=importers.FollowerAlgodImporter(sourcenet),
-        processors=[
-        ],
-        exporter=exporters.PostgresqlExporter(),
-    )
+class FollowerIndexerScenario(Scenario):
+    def __init__(self, sourcenet):
+        super().__init__(
+            name="follower_indexer_scenario",
+            importer=importers.FollowerAlgodImporter(sourcenet),
+            processors=[],
+            exporter=exporters.PostgresqlExporter(),
+        )
+
+    def get_validation_errors(self) -> list[str]:
```

Comment:
@shiqizng 
This breaks my promise not to modify existing tests. I found it convenient during development to add this validation and then modify it for the delete task. It might be useful to keep, or it may cause an occasional flake. If anyone expresses the view that it should be removed, I'll gladly remove.